### PR TITLE
Update Knative Serving references to 0.13.2.

### PR DESCRIPTION
**Pull request by markusthoemmes**
The title has it all. 0.13.2 has the latest and greatest.

### CODE REVIEW

**Comment by markusthoemmes** [comment]
/test 4.4-e2e-aws-ocp-44

Flake.

**Comment by markusthoemmes** [comment]
/retest

Checkout issues.

**Comment by markusthoemmes** [comment]
/retest

Setup issues.

**Comment by markusthoemmes** [comment]
/retest

A few more retries will fix the infra flakes... RIGHT?

**Comment by markusthoemmes** [comment]
/retest

Sigh...

**Comment by markusthoemmes** [comment]
/retest

Flakes...

**Comment by markusthoemmes** [comment]
That was a flake. Recorded here: https://issues.redhat.com/browse/SRVKS-510

File: hack/lib/vars.bash [hunk 1]
```diff
@@ -5,7 +5,7 @@ readonly BUILD_NUMBER=${BUILD_NUMBER:-$(uuidgen)}
 # shellcheck disable=SC1091,SC1090
 source "$(dirname "${BASH_SOURCE[0]}")/../../test/vendor/github.com/knative/test-infra/scripts/e2e-tests.sh"
 
-readonly KNATIVE_VERSION="${KNATIVE_VERSION:-v0.13.1}"
+readonly KNATIVE_VERSION="${KNATIVE_VERSION:-v0.13.2}"
```
Comment (matzew) [inline hack/lib/vars.bash:8]: shouldn't this be named `KNATIVE_SERVING` ? Since the the serverless operator installs now both: serving/eventing

Comment (markusthoemmes) [reply hack/lib/vars.bash:8]: presumably, yeah.

**Pull request merged**